src/components/events/UpcomingEvent.js

```diff
@@ -46,6 +46,7 @@ export default function UpcomingEvent({
         }}
         isOpen={showModal}
         closeModal={closeModal}
+        styles={{ backgroundColor: 'red' }}
```

Comment:
What was this change for?

Reply:
i used it for debugging but forgot to delete